kie-drools-wb/kie-drools-wb-distribution-wars/src/main/productized/webapp/logout.jsp

```diff
@@ -5,10 +5,13 @@
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ taglib uri="http://jakarta.apache.org/taglibs/i18n-1.0" prefix="i18n" %>
 <%
-  request.logout();
-  javax.servlet.http.HttpSession httpSession = request.getSession(false);
-  if (httpSession != null) {
-    httpSession.invalidate();
+  try {
+    request.logout();
+    javax.servlet.http.HttpSession httpSession = request.getSession(false);
+    if (httpSession != null) {
+      httpSession.invalidate();
+    }
+  } catch ( Exception e ) {
```

Comment:
I wonder, is it really necessary to just swallow the exception? I understand in some situations there is no better solution. If this is the case, could you please add there a comment explaining the reason?